runtime/gc_structs/ClassLocalInterfaceIterator.cpp

```diff
@@ -36,17 +36,15 @@
  * @return the next interface reference slot in the class
  * @return NULL if the class has no more interface references
  */
-J9Class **
+J9Class *
 GC_ClassLocalInterfaceIterator::nextSlot()
 {
-	J9Class **slotPtr;
-
-	if(_iTable == _superclassITable) {
-		return NULL;
+	J9Class *classPtr = NULL;
+	while (_iTable != _superclassITable) {
+		classPtr = _iTable->interfaceClass;
+		_iTable = _iTable->next;
+		if (NULL != classPtr) break;
```

Comment:
According Coding Standards `{` `}` brackets must be used even in case of single operator inside logical `if`